classes/import/marksessions.php

```diff
@@ -132,15 +132,15 @@ protected function get_column_data($row, $index) {
     /**
      * Constructor - parses the raw text for sanity.
      *
-     * @param string $text The raw csv text.
      * @param mod_attendance_structure $att The current assignment
+     * @param string $text The raw csv text.
      * @param string $encoding The encoding of the csv file.
      * @param string $delimiter The specified delimiter for the file.
      * @param string $importid The id of the csv import.
      * @param array $mappingdata The mapping data from the import form.
      * @param bool $useprogressbar Whether progress bar should be displayed, to avoid html output on CLI.
      */
-    public function __construct($text = null, $att, $encoding = null, $delimiter = null, $importid = 0,
+    public function __construct($att, $text = null, $encoding = null, $delimiter = null, $importid = 0,
```

Comment:
did you change the order of these params on purpose? - this doen'st seem to be a codechecker change?